docs/sphinx/source/installation.rst

```diff
@@ -191,9 +195,33 @@ if you make changes to pvlib during an interactive Python
 session (including a Jupyter notebook). Restarting the Python
 interpreter will also work.
 
-Remember to ``source activate pvlibdev`` (or whatever you named your
+Remember to ``conda activate pvlibdev`` (or whatever you named your
 environment) when you start a new shell or terminal.
 
+.. _compatibility:
+
+Compatibility
+-------------
+
+pvlib-python is compatible with Python versions 2.7 and 3.4-3.7.
+
+pvlib-python requires Pandas and Numpy. The minimum version requirements
+are specified in
+`setup.py <https://github.com/pvlib/pvlib-python/blob/master/setup.py>`_.
+They are typically releases from several years ago.
+
+A handful of pvlib-python features require additional packages that must
+be installed separately using pip or conda. These packages/features
+include:
+
+* scipy: single diode model, clear sky detection
+* pytables: Linke turbidity look up for clear sky models
+* numba: fastest solar position calculations
+* pyephem: solar positions calculations using an astronomical library
+* siphon: forecasting PV power using the pvlib.forecast module
```

Comment:
IMO having a `requirements.txt` file makes specifying requirements and letting users create non-conda virtual environments easier, here's my `requirements.txt` for pvlib in linux (not conda):
```
# build and run
pytz
numpy
scipy
pandas
numba
ephem
siphon
netcdf4
tables
# docs and testing
matplotlib
sphinx
sphinx_rtd_theme
ipython
jupyter
numpydoc
pytest
pytest-cov
pytest-mock
nose
nbsphinx
```
some of these are not available from either anaconda.org or conda-forge.

Reply:
I am open to 

1. adding a requirements file or files, and/or
2. adding support for e.g. ``pip install pvlib[all]``

in

1. this pull request, or
2. a separate pull request

Not sure what the best approach is, though, which makes me think defer each of those items to their own PRs. They are certainly related to this topic, though, especially if the result of the "Advanced User" or similar request involves a larger restructuring.

Reply:
Agreed let's consider `requirements.txt` in a separate PR, but I was just noting that there are more dependencies than just numpy and pandas.

Reply:
optional dependencies ;)